src/pages/index.js

```diff
@@ -188,8 +187,197 @@ const App = ({ data }) => (
+        <h1 id="our-story">Our Story — or how the Shyle came to be…</h1>
+        <h3>September, 2008</h3>
+        <p>Shannon and Kyle meet at college @ BYU</p>
+        <h3>October 31, 2008</h3>
+        <Img fluid={data.party.childImageSharp.fluid} />
+        <p>
+          <em>
+            The first known photo of Shannon and Kyle at a Halloween party
+          </em>
+        </p>
+        <h3>May, 2009</h3>
+        <p>Kyle and Shannon go on double date with… each other's roommates!</p>
+        <p>
+          Shannon says she wants to edit a science journal. Kyle thinks: “what
+          an attractive woman!”
+        </p>
+        <h3>July 2009</h3>
+        <Img fluid={data.garden.childImageSharp.fluid} />
+        <p>
+          <em>
+            Shannon hears that Kyle planted a garden behind his neighbor’s house
+            and asks to go see it. The garden is more impressive than she
+            expected.
+          </em>
+        </p>
+        <h3>August 2009</h3>
+        <p>
+          Kyle texts Shannon to ask her on a date and she doesn't respond for
+          several weeks because her phone is broken 😅
+        </p>
+        <p>They plan to go on another date, and then this happens:</p>
+        <p>
+          <strong>Shannon:</strong>
+          {` `}
+          “Kyle, I have to reschedule our date. I’m in the hospital with
+          appendicitis and go into surgery in 45 minutes.”
+        </p>
+        <p>
+          <strong>Kyle:</strong> “That is the best excuse I’ve ever heard for
+          rescheduling a date! Good luck!”
+        </p>
+        <p>
+          {" "}
+          They finally go on date a few days after Shannon has her appendix out.
+          Shannon was probably on pain killers. Neither of them remember the
+          events of the date.
+        </p>
+        <h3>October 2009</h3>
+        <p>Shannon exits to southern California for a 1.5 year LDS mission.</p>
+        <h3>January 2011</h3>
+        <p>Kyle exits to northern California to chase startup dreams.</p>
+        <h3>May 2015</h3>
+        <p>Shannon messages Kyle on Spotify</p>
+        <p>
+          It’s been 6 years since we last talked and then Shannon follows Kyle
+          on Spotify and thinks this song he's listening to is cool.
+        </p>
+        <iframe
+          src="https://open.spotify.com/embed/track/01txDiOnBPfMrVQwPQtDHR"
+          width="300"
+          height="380"
+          frameborder="0"
+          allowtransparency="true"
+          allow="encrypted-media"
+        />
+        <p>
+          Through Facebook, she realizes that Kyle lives in San Francisco and
+          since she’s already planning on road tripping through California with
+          a friend, she messages him on Facebook to see if he wants to hang out.
+        </p>
+        <p>
+          <strong>Shannon:</strong> Hey me and a friend are going to be in Palo
+          Alto/San Fran over 4th of July weekend! It'd be cool to meet up
```

Comment:
ok I'd take out our FB messages now that I think about it. They are just a bit long to read through.